lib/inferno/apps/cli/main.rb

```diff
@@ -52,6 +53,11 @@ def suites
       subcommand 'suite', Suite
 
       register(New, 'new', 'new TEST_KIT_NAME', 'Run `inferno new --help` for full help')
+
+      desc 'version', "Output Inferno core version (#{Inferno::VERSION})"
+      def version
+        puts "Inferno Core v#{Inferno::VERSION}"
```

Comment:
I think this difference would come up if you run `bundle exec` or not.

1. `bundle exec inferno version` -> runs inferno_core from Gemfile

2. `inferno version` -> runs inferno_core from last `gem install inferno_core` and ruby install

On machine if I do a fresh clone of inferno-template I get:
<img width="866" alt="image" src="https://github.com/inferno-framework/inferno-core/assets/8866137/8d61863f-ea13-490a-9631-8864036d9ffa">

If people start using `inferno new` instead of git clone this issue will be somewhat allevaited, the versions will at least match until a new inferno version is published and either `gem update --system` or `bundle update` is executed but not both.

[Rails loads bundler in its boot.rb](https://guides.rubyonrails.org/initialization.html#launch-bang), but I think Inferno can't do that because we need foreman, and foreman can't be added to the gemspec. Same reason we have to run `bundle exec inferno services` and then `inferno start` without bundle. So I guess one solution would be replacing or in-housing foreman functionality.

For now only thing I can think of is adding this version command and help devs figure out any version mismatch.